src/libpriv/rpmostree-bwrap.c

```diff
@@ -313,18 +368,27 @@ rpmostree_bwrap_new (int rootfs_fd,
   va_list args;
   switch (mutable)
     {
-    case RPMOSTREE_BWRAP_IMMUTABLE:
-      rpmostree_bwrap_append_bwrap_argv (ret, "--ro-bind", "usr", "/usr", NULL);
-      break;
     case RPMOSTREE_BWRAP_MUTATE_ROFILES:
-      if (!setup_rofiles_usr (ret, error))
-        return NULL;
+      {
+        if (!setup_rofiles_usr (ret, error))
+          return NULL;
+        rpmostree_bwrap_append_bwrap_argv (ret, "--ro-bind", "./var", "/var", NULL);
```

Comment:
Hmm, my vote would be for keeping this switch strictly about `/usr` and letting callers specify the additional binds for `/var` and `/etc`, as before. My reasoning is that it's not immediately obvious why e.g. `MUTATE_ROFILES` implies `--ro-bind ./var /var`. Those really feel like details specific to the context from which we were called, and hardcoding this magic here makes the code harder to read.